releasenotes/notes/experiments-hgp-88b2416769125e38.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    The method
+    :meth:`qiskit.providers.ibmq.experiment.ExperimentService.experiments`
```

Comment:
Should be `qiskit.providers.ibmq.experimentservice.ExperimentService.experiments`.